incubator/metacontroller/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Default values for helmcontroller.
+image:
+  repository: metacontroller/metacontroller
+  tag: 0.2
+
+rbac:
+  create: true
+
+serviceAccount:
+  create: true
+  name: metacontroller
```

Comment:
See https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md for the name and other rbac info